api/tests/routes/pro/get_venue_offers_statistics_test.py

```diff
@@ -0,0 +1,163 @@
+        offers_factories.OfferFactory.create_batch(3, venue=venue, validation=OfferValidationStatus.PENDING)
+
+        # 2 pending collective offers + 2 pending collective offer template = 4 pending collective offers
+        # 1 of each rejected to ensure they are not counted
+        educational_factories.create_collective_offer_by_status(
+            CollectiveOfferDisplayedStatus.UNDER_REVIEW, venue=venue
+        )
+        educational_factories.create_collective_offer_by_status(
+            CollectiveOfferDisplayedStatus.UNDER_REVIEW, venue=venue
+        )
+        educational_factories.create_collective_offer_by_status(CollectiveOfferDisplayedStatus.REJECTED, venue=venue)
+        educational_factories.create_collective_offer_template_by_status(
+            CollectiveOfferDisplayedStatus.UNDER_REVIEW, venue=venue
+        )
+        educational_factories.create_collective_offer_template_by_status(
+            CollectiveOfferDisplayedStatus.UNDER_REVIEW, venue=venue
+        )
+        educational_factories.create_collective_offer_template_by_status(
+            CollectiveOfferDisplayedStatus.REJECTED, venue=venue
+        )
+
+        client = client.with_session_auth(user_offerer.user.email)
+
+        with testing.assert_num_queries(self.num_queries):
+            response = client.get(f"/venue/{venue.id}/offers-statistics")
+            assert response.status_code == 200
+
+        assert response.json == {
+            "publishedPublicOffers": 1,
+            "publishedEducationalOffers": 2,
+            "pendingPublicOffers": 3,
+            "pendingEducationalOffers": 4,
+        }
+
+    def test_get_offers_stats_with_no_public_offers(self, client):
+        user_offerer = offerers_factories.UserOffererFactory()
+        venue = offerers_factories.VenueFactory(managingOfferer=user_offerer.offerer)
+        educational_factories.PublishedCollectiveOfferFactory.create_batch(2, venue=venue)
+        educational_factories.UnderReviewCollectiveOfferFactory.create_batch(2, venue=venue)
+
+        client = client.with_session_auth(user_offerer.user.email)
+        venue = user_offerer.offerer.managedVenues[0]
+        with testing.assert_num_queries(self.num_queries):
+            response = client.get(f"/venue/{venue.id}/offers-statistics")
+            assert response.status_code == 200
+
+        assert response.json == {
+            "publishedPublicOffers": 0,
+            "publishedEducationalOffers": 2,
+            "pendingPublicOffers": 0,
+            "pendingEducationalOffers": 2,
+        }
+
+    def test_get_offers_stats_with_no_collective_offers(self, client):
+        user_offerer = offerers_factories.UserOffererFactory()
+        venue = offerers_factories.VenueFactory(managingOfferer=user_offerer.offerer)
+        for _ in range(2):
+            offer = offers_factories.OfferFactory(venue=venue, validation=OfferValidationStatus.APPROVED)
+            offers_factories.StockFactory(offer=offer)
```

Comment:
C'est du détail mais est-ce qu'on ne pourrait pas simplifier cette boucle ? 

```suggestion
        offers_factories.StockFactory.create_batch(2, offer__venue=venue, offer__validation=OfferValidationStatus.APPROVED)
```